src/agora/agora.h

```diff
@@ -15,6 +15,7 @@
 #include <system_error>
 #include <vector>
 
```

Comment:
In general, You have a lot of structs to group items.   Per coding standards these should move to 1 file per each definition.   However, since they are so simple it might be reasonable to move the as a group to 1 file.   We need to namespace these, but I don't want to go crazy with the namespacing.  Just pick something simple for now.  

Brings up another point.   I understand why you "structed" these values.  But it would be best if you made small classes out of them (same memory footprint).  And added accessors / modifiers for safer accesses.

Reply:
Yes, I agree. I'll create a small class of these structs, define them in `common/buffer.h`. Then, the rest of the functions in `agora_helper.h` can be migrated to `utils.cc`, thus deleting `agora_helper.h`.